test/snippet/argument_parser/argument_parser_2.cpp

```diff
@@ -13,7 +13,7 @@ int main(int argc, char ** argv)
     {
         myparser.parse();
     }
-    catch (seqan3::parser_invalid_argument const & ext) // the user did something wrong
+    catch (seqan3::invalid_argument const & ext) // the user did something wrong
```

Comment:
Please change this to `argument_parser_exception` and change the comment behind it to
`// catch all other exceptions caused by user errors`.
This goes for all snippets in the argument_parser directory and subfolders.